packages/Input/src/input.css

```
@@ -0,0 +1,118 @@
+@import "~@igloo-ui/tokens/dist/variables.css";
+@import "~@igloo-ui/tokens/dist/fonts.css";
+
+:root {
+  /* Default */
+  --ids-input-height: var(--space-6);
+  --ids-input-font-family: var(--primary-font-family);
+  --ids-input-font-weight: var(--font-weight-regular);
+  --ids-input-padding: var(--space-3);
+  --ids-input-font-size: var(--font-size-4);
+  --ids-input-line-height: var(--space-4);
+  --ids-input-border-radius: var(--border-radius-sm);
+  --ids-input-border-color: var(--grey-300);
+  --ids-input-background-color: var(--samoyed);
+  --ids-input-margin: var(--space-1) var(--space-0);
+
+  /* Small */
+  --ids-input-height-small: var(--space-5);
+  --ids-input-font-size-small: var(--font-size-3);
+  --ids-input-line-height-small: var(--line-height-xxl);
+  --ids-input-letter-spacing-small: var(--font-weight-regular);
+
+  /* Placeholder */
+  --ids-input-color-placeholder: var(--grey-400);
+
+  /* Hover */
+  --ids-input-border-color-hover: var(--grey-500);
+
+  /* Focus */
+  --ids-input-border-color-focus: var(--electric-blue-500);
+
+  /* Readonly */
+  --ids-input-color-readonly: var(--grey-500);
+  --ids-input-background-color-readonly: var(--grey-250);
```

idem, pas de `-color`
```suggestion
  --ids-input-background-readonly: var(--grey-250);
```